pkg/pkg.mk

```diff
@@ -31,7 +31,7 @@ endif
 $(PKG_BUILDDIR)/.git-downloaded:
 	rm -Rf $(PKG_BUILDDIR)
 	mkdir -p $(PKG_BUILDDIR)
-	$(GITCACHE) clone "$(PKG_URL)" "$(PKG_VERSION)" "$(PKG_BUILDDIR)"
+	GIT_CACHE_DIR=$(GIT_CACHE_DIR) $(GITCACHE) clone "$(PKG_URL)" "$(PKG_VERSION)" "$(PKG_BUILDDIR)"
```

Comment:
Do you need to write this environment variable on the command line if it is already exported (vars.mk)?

Reply:
That is a remnant of when I tried to not export the variable, but it failed because of sub makefiles.

I left it to show that the script was using the variable but I can remove it.

Reply:
I removed it, it is an internal script anyway and `git grep GIT_CACHE_DIR` finds it.